applicationset/generators/git.go

```diff
@@ -58,12 +61,36 @@ func (g *GitGenerator) GenerateParams(appSetGenerator *argoprojiov1alpha1.Applic
 
 	noRevisionCache := appSet.RefreshRequired()
 
+	var project string
+	if strings.Contains(appSet.Spec.Template.Spec.Project, "{{") {
+		project = appSetGenerator.Git.Template.Spec.Project
+	} else {
+		project = appSet.Spec.Template.Spec.Project
+	}
+
+	log.Info("===============project=================")
+	log.Info(project)
+
+	appProject := &argoprojiov1alpha1.AppProject{}
+	if err := client.Get(context.TODO(), types.NamespacedName{Name: appSet.Spec.Template.Spec.Project, Namespace: appSet.Namespace}, appProject); err != nil {
+		return nil, fmt.Errorf("error getting project %s: %w", project, err)
+	}
+
+	log.Infof("===============GPG Enabled : %v=================", gpg.IsGPGEnabled())
```

Comment:
i would not do such log, can we just replace it to some plain text?

Reply:
I thought I removed this log statements.. I will remove them again.